front/lib/api/assistant/jit_actions.ts

```diff
@@ -370,3 +408,43 @@ export async function renderConversationForModelJIT({
     tokensUsed,
   });
 }
+
+async function listConversationFiles({
```

Comment:
This method is kind of lost on its own while it would be the action `run` method in the case of a fully typed JITListFiles....